src/background/responders.ts

```diff
@@ -216,32 +210,31 @@ export const responders: { [T in Action['type']]: Responder<T> } = {
 
     return {
       tabs: nextTabs,
-      alert: action.payload.error.message,
+      alert: error.message,
     }
   },
-  'chrome.windows.getAll'(state, action): Partial<AppState> {
+  'chrome.windows.getAll'(state, { windows }): Partial<AppState> {
     return {
-      focusedWindowId: action.payload.windows.find(win => win.focused)!.id,
+      focusedWindowId: windows.find(win => win.focused)!.id,
     }
   },
-  'chrome.tabs.query'(state, action): Partial<AppState> {
+  'chrome.tabs.query'(state, payload): Partial<AppState> {
```

Comment:
We're not using `payload` for most reponders. Let's use destructuring here and everywhere else in the code.